lib/httplib/httpheaders.go

```diff
@@ -210,31 +210,6 @@ func SetIndexContentSecurityPolicy(h http.Header, cfg proto.Features, urlPath st
 	h.Set("Content-Security-Policy", cspString)
```

Comment:
Not be strictly related to this PR, but you should consider implementing a report-uri.